www/data/sql_funcs.json

```diff
@@ -97,6 +97,14 @@
             {
                 "signature": "mod(x: N, y: N) -> N",
                 "description": "`x % y`"
+            },
+            {
+                "signature": "round(x: N) -> N",
+                "description": "Equivalent to `floor(x)`"
```

Comment:
It is not equivalent to floor(x) because round(1.7) -> 2. Just round to the nearest whole number.

Reply:
What a derp. Sorry, folks.